### ISSUE TITLE
Bug: Type-error reading `wordsToOverrideDict`

### ISSUE THREAD

**Issue by chrisgrieser** [artisticat1/nl-syntax-highlighting#1]
for some reasons, I do not get any highlighting at all. I looked in the console and saw this error:

```js
app.js:1 TypeError: Cannot read properties of undefined (reading 'wordsToOverrideDict')
    at getDecosOnLine (plugin:nl-syntax-highlighting:16511:53)
    at getAllDecosByLine (plugin:nl-syntax-highlighting:16481:21)
    at new import_view.ViewPlugin.fromClass.decorations (plugin:nl-syntax-highlighting:16536:30)
    at e.create (app.js:1:350672)
    at e.update (app.js:1:351102)
    at e.ensureHandlers (app.js:1:378274)
    at e.updatePlugins (app.js:1:443716)
    at e.update (app.js:1:441962)
    at e._dispatch (app.js:1:438758)
    at e.dispatch (app.js:1:440536)
```

I tried using the plugin with the setting "Words to override" empty and filled with the example entries, in both cases I get the same error above.

**Comment by artisticat1** [comment]
Apologies! I made a mistake when renaming this plugin's ID. Should be fixed in 0.3.0.

**Comment by chrisgrieser** [comment]
no worries, thanks for the quick fix!
